examples/advanced/bionemo/downstream/sabdab/run_sim_sabdab.py

```diff
@@ -25,7 +25,7 @@
 from nvflare.job_config.script_runner import BaseScriptRunner
 
 sys.path.append(os.path.join(os.getcwd(), ".."))  # include parent folder in path
-from bionemo_filters import BioNeMoParamsFilter
+from bionemo_filters import BioNeMoParamsFilter, BioNeMoStateDictFilter
```

Comment:
@holgerroth can you check if you have these filters in the main branch? thanks

Reply:
yes, they are now in the main branch. https://github.com/NVIDIA/NVFlare/blob/main/examples/advanced/bionemo/downstream/bionemo_filters.py